### PR TITLE
Update docs request template to request that people add docs issues t…

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
- This adds a CTA when creating a docs issue to add it to the GH Docs Roadmap project

## Misc Changes
- Improves some of the language around docs issues

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -6,7 +6,12 @@ labels: type/enhancement, area/documentation
...
+It can be as pedantic as a typo or as titanic as an entire tutorial request.
 
-## Tell us about the documentation you'd like us to add or update
 
-## Are there any related areas of documentation that currently exist? Could you point to them? 
+### If applicable, add links to the relevant docs that should be updated
+- [Link to relevant docs page](https://docs.airbyte.io)
+
+### Finally, add this issue to the `Documentation Roadmap` GitHub project on the right ->
+This helps us prioritize your issue and will make sure it's done quickly!
```
Comment (marcosmarxm) [inline .github/ISSUE_TEMPLATE/documentation.md:17]: I think contributors can modify this themselves.
We can improve this script https://github.com/airbytehq/airbyte/blob/master/.github/workflows/project.yaml

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/documentation.md (hunk 1 above)
Comment (avaidyanatha) [reply .github/ISSUE_TEMPLATE/documentation.md:17]: Oh we do have the github action auto assign, ok I'll throw it in there.

Comment (avaidyanatha) [reply .github/ISSUE_TEMPLATE/documentation.md]: Ok @marcosmarxm done!

**Comment by marcosmarxm** [comment]
i think you need to run `./gradlew format` because is modifying a project file

**Review by marcosmarxm — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

**Pull request merged**